src/app/Main/Body/QueryPage/Results/Content-Results.component.tsx

```diff
@@ -106,13 +108,17 @@ export function ContentResultsComponent(): JSX.Element {
 
     const fetchIsolateCounted = async (): Promise<void> => {
         const isolateCountResponse: Response = await fetch(ISOLATE_COUNT_URL);
-        const isolateCountProp: IsolateCountedDTO = await isolateCountResponse.json();
+        const isolateCountStatus = isolateCountResponse.status;
+        setCountedStatus(isolateCountStatus);
 
-        getTableContext(isolateCountProp);
-        setData({
-            ...data,
-            nrOfSelectedIsolates: isolateCountProp.totalNumberOfIsolates,
-        });
+        if (isolateCountStatus === 200) {
+            const isolateCountProp: IsolateCountedDTO = await isolateCountResponse.json();
+            getTableContext(isolateCountProp);
+            setData({
+                ...data,
+                nrOfSelectedIsolates: isolateCountProp.totalNumberOfIsolates,
+            });
+        }

```

Comment:
The additional status checking and errorPage loading here is another reason why it is better to place the fetching stuff in the container (as stated in a previous commit)

Reply:
This will change with the refactoring of the container-view-structure.